src/components/alerts/Banner/Banner.tsx

```diff
@@ -47,13 +48,36 @@ export default class Banner extends React.Component<IProps> {
 			return null;
 		}
 
+		let color: ButtonPropColor = ButtonPropColor.gray;
```

Comment:
Is it worth changing this variable to a more descriptive name such as `buttonColor`?

Reply:
Would you clarify what you mean by `more descriptive`?

Reply:
Yes, looking at it again, I read you. I'll change the variable name to be clearer. Thanks!